package.json

```diff
@@ -31,18 +32,25 @@
   },
   "devDependencies": {
     "@istanbuljs/esm-loader-hook": "^0.1.2",
+    "@tsconfig/node16": "^1.0.2",
+    "@types/http-errors": "^1.8.1",
+    "@types/node": "^16.11.11",
+    "@types/sinon": "^10.0.6",
+    "@types/tap": "^15.0.5",
+    "@typescript-eslint/eslint-plugin": "^5.5.0",
+    "@typescript-eslint/parser": "^5.5.0",
     "eslint": "^8.4.0",
     "eslint-plugin-sql": "^2.0.0",
     "postgrator-cli": "^4.0.0",
     "prettier": "^2.5.1",
     "sinon": "^12.0.1",
-    "tap": "^15.1.5"
+    "tap": "^15.1.5",
+    "ts-node-dev": "^1.1.8",
+    "typescript": "^4.5.2"
   },
   "tap": {
     "node-arg": [
-      "--no-warnings",
```

Comment:
Those arguments there no effect on the current tests, so I have removed them since the `esm-loader-hook` doesn't works with typescript. 

See: https://github.com/TypeStrong/ts-node/issues/935